### PR TITLE
Update CHANGELOG for 0.8.0

### PR DESCRIPTION
**Pull request by angelozerr**
Update CHANGELOG for 0.8.0

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,62 @@
 # Change Log
 
+## [0.8.0](https://github.com/angelozerr/lsp4xml/milestone/9?closed=1) (July 23, 2019)
+
+### Enhancements
+
+ * Find definition for start/end tag element. See[#535](https://github.com/angelozerr/lsp4xml/issues/535)
+ * Improve Source information from completion. See[#529](https://github.com/angelozerr/lsp4xml/issues/529)
```
Comment (xorye) [inline CHANGELOG.md]: Improve source information from completion.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
+ * `XSD CodeLens` references should be clickable. See[#490](https://github.com/angelozerr/lsp4xml/issues/490)
```
Comment (xorye) [inline CHANGELOG.md]: Can we say "Add support for clickable XSD CodeLens" instead? I'm personally don't think it should say "should be clickable".

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,62 @@
...
+ * Add support for `textDocument/documentHighlight` for XML Schema types. See[#470](https://github.com/angelozerr/lsp4xml/issues/470)
+ * Add support for `textDocument/completion` for xs:element/@name / xs:extension/@base. See[#451](https://github.com/angelozerr/lsp4xml/issues/451)
+ * Support selective enablement of outline per file. See[#427](https://github.com/angelozerr/lsp4xml/issues/427)
```
Comment (xorye) [inline CHANGELOG.md]: I would suggest: "Add support for selective outline enablement per file"

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,62 @@
...
+ * `.ent` and `.mod` files should be parsed as DTDs. See[#380](https://github.com/angelozerr/lsp4xml/issues/380)
```
Comment (xorye) [inline CHANGELOG.md]: I would suggest "Parse `.ent` and `.mod` files as DTD files"

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,62 @@
...
+ * Add support for `textDocument/typeDefinition` from XML to XMLSchema/DTD . See[#371](https://github.com/angelozerr/lsp4xml/issues/371)
+ * Show possible elements for completion based on XML Schema. See[#347](https://github.com/angelozerr/lsp4xml/issues/347)
```
Comment (xorye) [inline CHANGELOG.md]: I would suggest "Show relevant XML completion options based on XML Schema"

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,62 @@
...
+ * Add support for `textDocument/definition` for XML Schema. See[#148](https://github.com/angelozerr/lsp4xml/issues/148)
+ * Add support for `textDocument/references` for XML Schema types. See[#58](https://github.com/angelozerr/lsp4xml/issues/58)
+ * Add support for `textDocument/codelens` for XML Schema types. See[#55](https://github.com/angelozerr/lsp4xml/issues/55)
+ * Add completion for `comment` and `#region`. See[#54](https://github.com/angelozerr/lsp4xml/issues/54)
+ * CData block auto-complete. See[#168](https://github.com/redhat-developer/vscode-xml/issues/168)
```
Comment (xorye) [inline CHANGELOG.md]: I would suggest "Add completion for CData block"

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,62 @@
...
+ * Documentation in completion has monospace font. See[#32](https://github.com/redhat-developer/vscode-xml/issues/32)
```
Comment (xorye) [inline CHANGELOG.md]: I would suggest: "Add markdown support for hover text and completion descriptions"

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,62 @@
...
+
+### Bug Fixes
+
+ * Hover markup kind response ignores the hover client capability. See[#525](https://github.com/angelozerr/lsp4xml/issues/525)
```
Comment (xorye) [inline CHANGELOG.md]: I would suggest "Hover markup response ignored the hover client capability"

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,62 @@
...
+ * Loss of completion capability from the client. See[#522](https://github.com/angelozerr/lsp4xml/issues/522)
```
Comment (xorye) [inline CHANGELOG.md]: Completion capability was lost in specific scenarios

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,62 @@
...
+ * Error range for `EntityNotDeclared`. See[#518](https://github.com/angelozerr/lsp4xml/issues/518)
```
Comment (xorye) [inline CHANGELOG.md]: Fixed error range

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,62 @@
...
+ * Improve revalidate XML files when XSD files is saved. See[#506](https://github.com/angelozerr/lsp4xml/issues/506)
```
Comment (xorye) [inline CHANGELOG.md]: Improved XML revalidation when XSD files are saved

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,62 @@
...
+ * Error range for `src-import.1.2`. See[#499](https://github.com/angelozerr/lsp4xml/issues/499)
```
Comment (xorye) [inline CHANGELOG.md]: Fixed error range

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,62 @@
...
+ * Error range for `s4s-elt-invalid-content.3`. See[#496](https://github.com/angelozerr/lsp4xml/issues/496)
+ * NPE in `textDocument/definition` in XSD files. See[#488](https://github.com/angelozerr/lsp4xml/issues/488)
```
Comment (xorye) [inline CHANGELOG.md]: Fixed NPE in

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,5 +1,62 @@
...
+ * Error range for `AttributePrefixUnbound`. See[#476](https://github.com/angelozerr/lsp4xml/issues/476)
+ * Error range for `EmptyTargetNamespace`. See[#472](https://github.com/angelozerr/lsp4xml/issues/472)
+ * Error range for `ct-props-correct.3`. See[#467](https://github.com/angelozerr/lsp4xml/issues/467)
+ * Error range for `sch-props-correct.2`. See[#462](https://github.com/angelozerr/lsp4xml/issues/462)
+ * Error range for `s4s-elt-must-match.2`. See[#458](https://github.com/angelozerr/lsp4xml/issues/458)
+ * Error range for `ct-props-correct.3`. See[#455](https://github.com/angelozerr/lsp4xml/issues/455)
+ * Error range for `src-ct.1`. See[#453](https://github.com/angelozerr/lsp4xml/issues/453)
+ * Error range for `duplicate attribute`.. See[#452](https://github.com/angelozerr/lsp4xml/issues/452)
+ * Error range for `p-props-correct.2.1`. See[#436](https://github.com/angelozerr/lsp4xml/issues/436)
+ * Element/Attribute names must be case sensitive. See[#433](https://github.com/angelozerr/lsp4xml/issues/433)
```
Comment (xorye) [inline CHANGELOG.md]: Fixed case sensitivity problems for element and attribute names

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,5 +1,62 @@
...
+ * Selection formatting ignores attribute indentation preference. See[#429](https://github.com/angelozerr/lsp4xml/issues/429)
+ * Error range for `cos-all-limited.2`. See[#428](https://github.com/angelozerr/lsp4xml/issues/428)
+ * Error range for `src-element.3`. See[#420](https://github.com/angelozerr/lsp4xml/issues/420)
+ * Documents with an Internal Subset DOCTYPE shouldn't stop trying to bind. See[#379](https://github.com/angelozerr/lsp4xml/issues/379)
```
Comment (xorye) [inline CHANGELOG.md]: Documents with an Internal Subset DOCTYPE had stopped trying to bind.

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,5 +1,62 @@
...
+ * Discrepancy in completion between prefixed and default namespaces. See[#311](https://github.com/angelozerr/lsp4xml/issues/311)
```
Comment (xorye) [inline CHANGELOG.md]: Fixed discrepancy in completion between prefixed and default namespaces.

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,5 +1,62 @@
...
+ * When DTD is not found, XML is not checked for well-formedness. See[#167](https://github.com/redhat-developer/vscode-xml/issues/167)
```
Comment (xorye) [inline CHANGELOG.md]: XML did not validate when bounded DTD file was not found

File: CHANGELOG.md [hunk 18]
```diff
@@ -1,5 +1,62 @@
...
+ * Formatter inserts spaces in empty lines. See[#157](https://github.com/redhat-developer/vscode-xml/issues/157)
+ * vscode not reloading and revalidating XML/XSD files modified outside itself. See[#131](https://github.com/redhat-developer/vscode-xml/issues/131)
```
Comment (xorye) [inline CHANGELOG.md]: Client was not revalidating XML files when relevant XSD files were modified outside the client.

File: CHANGELOG.md [hunk 19]
```diff
@@ -1,5 +1,62 @@
...
+
+### Performances
```
Comment (xorye) [inline CHANGELOG.md]: Performance

File: CHANGELOG.md [hunk 20]
```diff
@@ -1,5 +1,62 @@
...
+
+ * Improve XML Scanner performance. See[#444](https://github.com/angelozerr/lsp4xml/issues/444)
+ * Use CompletableFuture to load DOMDocument. See[#439](https://github.com/angelozerr/lsp4xml/issues/439)
+ * Feedback from memory. See[#438](https://github.com/angelozerr/lsp4xml/issues/438)
```
Comment (xorye) [inline CHANGELOG.md]: I would suggest "Examined memory usage"

File: CHANGELOG.md [hunk 21]
```diff
@@ -1,5 +1,62 @@
...
+ * Improve performance of TextDocument update (in async) with TreeLineTracker. See[#426](https://github.com/angelozerr/lsp4xml/issues/426)
```
Comment (xorye) [inline CHANGELOG.md]: I would suggest "Improved `TextDocument` update (in async) performance with `TreeLineTracker`."

File: CHANGELOG.md [hunk 22]
```diff
@@ -1,5 +1,62 @@
...
+ * Test Large Files for Performance. See[#48](https://github.com/angelozerr/lsp4xml/issues/48)
```
Comment (xorye) [inline CHANGELOG.md]: Tested large files for performance.

File: CHANGELOG.md (hunk 11 above)
Comment (xorye) [reply CHANGELOG.md]: But I think this might be more of an enhancement rather than a bug fix?

File: CHANGELOG.md (hunk 1 above)
Comment (angelozerr) [reply CHANGELOG.md]: @xorye updated

File: CHANGELOG.md (hunk 3 above)
Comment (angelozerr) [reply CHANGELOG.md]: @xorye updated

File: CHANGELOG.md (hunk 6 above)
Comment (angelozerr) [reply CHANGELOG.md]: @xorye updated

File: CHANGELOG.md (hunk 7 above)
Comment (angelozerr) [reply CHANGELOG.md]: @xorye updated

File: CHANGELOG.md (hunk 8 above)
Comment (angelozerr) [reply CHANGELOG.md]: @xorye updated

File: CHANGELOG.md (hunk 10 above)
Comment (angelozerr) [reply CHANGELOG.md]: @xorye updated

File: CHANGELOG.md (hunk 12 above)
Comment (angelozerr) [reply CHANGELOG.md]: @xorye updated

File: CHANGELOG.md (hunk 14 above)
Comment (angelozerr) [reply CHANGELOG.md]: @xorye updated

**Comment by angelozerr** [comment]
@xorye done

File: CHANGELOG.md [hunk 23]
```diff
@@ -1,5 +1,63 @@
...
+ * Add completion for `CDATA` block. See [#168](https://github.com/redhat-developer/vscode-xml/issues/168)
+ * Find definition for start/end tag element. See [#535](https://github.com/angelozerr/lsp4xml/issues/535)
+ * Show `relevant XML` completion options based on XML Schema. See [#347](https://github.com/angelozerr/lsp4xml/issues/347) 
+ * Improve `XSD source` information for XML completion. See [#529](https://github.com/angelozerr/lsp4xml/issues/529)
+ * Add support for `textDocument/documentHighlight` for XML Schema types. See [#470](https://github.com/angelozerr/lsp4xml/issues/470)
+ * Add support for `textDocument/completion` for xs:element/@name / xs:extension/@base. See [#451](https://github.com/angelozerr/lsp4xml/issues/451)
+ * Add support for selective outline enablement per file. See [#427](https://github.com/angelozerr/lsp4xml/issues/427)
+ * Parse `.ent` and `.mod` files as DTD files. See [#380](https://github.com/angelozerr/lsp4xml/issues/380)
+ * Add support for `textDocument/typeDefinition` from XML to XMLSchema/DTD . See [#371](https://github.com/angelozerr/lsp4xml/issues/371)
```
Comment (xorye) [inline CHANGELOG.md]: XMLSchema/DTD.

File: CHANGELOG.md [hunk 24]
```diff
@@ -1,5 +1,63 @@
...
+ * Formatter inserts spaces in empty lines. See [#157](https://github.com/redhat-developer/vscode-xml/issues/157)
+ * VSCode was not revalidating XML files when relevant XSD files were modified outside VSCode. See [#131](https://github.com/redhat-developer/vscode-xml/issues/131)
+
+### Performance
+
+ * Improve XML Scanner performance. See [#444](https://github.com/angelozerr/lsp4xml/issues/444)
+ * Use CompletableFuture to load DOMDocument. See [#439](https://github.com/angelozerr/lsp4xml/issues/439)
+ * Examined memory usage. See [#438](https://github.com/angelozerr/lsp4xml/issues/438)
+ * Improved `TextDocument` update (in async) performance with `TreeLineTracker`. See [#426](https://github.com/angelozerr/lsp4xml/issues/426)
+ * TTested large files for performance. See [#48](https://github.com/angelozerr/lsp4xml/issues/48)
```
Comment (xorye) [inline CHANGELOG.md]: Tested

**Comment by angelozerr** [comment]
@xorye it should be done

**Pull request merged**